src/Sidekick.Apis.Poe/Trade/TradeFilterService.cs

```diff
@@ -42,12 +42,8 @@ public IEnumerable<PseudoModifierFilter> GetPseudoModifierFilters(Item item)
         }
     }
 
-    public async Task<PropertyFilters> GetPropertyFilters(Item item)
+    public PropertyFilters GetPropertyFilters(Item item) => new()
```

Comment:
As said in the interface comment, revert the changes here. We have things cooking.